libraries/botbuilder/tests/channelServiceRoutes.test.js

```diff
@@ -29,25 +29,37 @@ class MockResponse {
     }
 }
 
-describe('channelServiceRoutes', function() {
+describe('channelServiceRoutes', function () {
```

Comment:
These changes are orthogonal to the PR but I cleaned up some non-sandboxed `sinon` usage and formatted the file.